src/openmrs-backend-dependcies.ts

```diff
@@ -0,0 +1,49 @@
+import { openmrsFetch } from "@openmrs/esm-api";
+const semver = require("semver");
```

Comment:
`require()` is for node code. But esm-devtools is browser code. We should use `import` instead of  `require` for browser code.

```suggestion
import * as semver from 'semver';
```